packages/medical-domain/src/domains/repositories/statistics/insulin-statistics.service.ts

```diff
@@ -100,12 +102,32 @@ function getTotalInsulinAndWeightData(basalsData: Basal[], bolusData: Bolus[], n
   }
 }
 
+function getAutomatedAndManualBasalDuration(basalsData: Basal[], numDays: number, dateFilter: DateFilter): TimeInAutoStatistics {
```

Comment:
```suggestion
function getAutomatedAndManualBasalDuration(basalsData: Basal[], dateFilter: DateFilter): TimeInAutoStatistics {
```
numDays seems useless